src/test/unit/StrategyManagerUnit.t.sol

```diff
@@ -2167,12 +2245,46 @@ contract StrategyManagerUnitTests is Test {
         return (queuedWithdrawal, tokensArray, withdrawalRoot);
     }
 
+    function _setUpQueuedWithdrawalStructSingleStratMultipleStrategies(
```

Comment:
what is this naming?
is this supposed to be `_setUpQueuedWithdrawalStruct_MultipleStrategies` or something?